bip-txrelayv2.mediawiki

```diff
@@ -0,0 +1,49 @@
+<pre>
+  BIP: XXX
+  Layer: Peer Services
+  Title: Transaction Relay V2
+  Author: Antoine Riard <btc@ariard.me>
+  Comments-Summary: No comments yet.
+  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-0339
```

Comment:
```suggestion
```

Reply:
Corrected with `BIP-0XXX`, in my memory bip339 is the oldest bip related to p2p tx-relay upgrade (as bip338 was never implemented and deployed). So I copy past bip layout from it.